app/knight_class.py

```diff
@@ -0,0 +1,16 @@
+class Knight:
+    def __init__(self,
```

Comment:
After expanding the brackets, write all arguments on a new line, including the first one